src/coreclr/vm/methodtable.h

```diff
@@ -3342,6 +3342,7 @@ public :
         // As you change the flags in WFLAGS_LOW_ENUM you also need to change this
         // to be up to date to reflect the default values of those flags for the
         // case where this MethodTable is for a String or Array
+        // TODO, AndrewAu, this looks important, do it!
```

Comment:
Since you are not adjusting the set of flags, you should not need to change anything here.